app/helpers/application_helper.rb

```diff
@@ -6,8 +6,7 @@ module ApplicationHelper
   include ApplicationAnnouncementsHelper
   include ApplicationWidgetsHelper
   include ApplicationCocoonHelper
-
-  include ApplicationNotificationsConcern
```

Comment:
hmm... this whitespace is used to separate helpers and the concern before. But I made a mistake that all things should be helpers. So this whitespace is useless...